lib/improver/metadata/utilities.py

```diff
@@ -44,16 +42,14 @@ def generate_hash(data_in):
     Args:
         data_in (any):
             The data from which a hash is to be generated. This can be of any
-            type that can be pickled.
+            type that can be dumped to JSON (dict, list, str, float, etc).
```

Comment:
This docstring refers to dumping to JSON, but the code uses prettyprint which is more general (it'll use __repr__ methods of non built-in classes). Suggest changing either the docstring or the code so that these match.

Reply:
Done, gone for prettyprinting